Makefile

```diff
@@ -318,11 +318,15 @@ create/olm/bundle:
 
 .PHONY: create/3scale/index
 create/3scale/index:
-	./scripts/create-3scale-index.sh
+	PRODUCT=3scale ./scripts/create-3scale-index.sh
```

Comment:
Maybe we need more generic name of the script? like `create-index-image.sh` or something like this?
PRODUCT=3scale ./scripts/create-3scale-index.sh
PRODUCT=rhsso ./scripts/create-3scale-index.sh
PRODUCT=mcg ./scripts/create-3scale-index.sh